src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int LOTTERY_MAX_COUNT = 100;
+    private final Random random = new Random();
+    ColorSupplier supplier = new ColorSupplier();

```

Comment:
Declare the `supplier` variable as private and final to follow encapsulation and immutability principles. Use `private final ColorSupplier supplier = new ColorSupplier();` instead. 🚀